src/bundle/Resources/views/admin/content_type/view.html.twig

```diff
@@ -2,6 +2,8 @@
 
 {% trans_default_domain 'content_type' %}
 
+{% block bodyClass %}ez-content-view{% endblock %}
```

Comment:
This `bodyClass` should reflect where you are in the admin panel (this is done to have the context in CSS).
`ez-content-view` means that you are in the `content structure`